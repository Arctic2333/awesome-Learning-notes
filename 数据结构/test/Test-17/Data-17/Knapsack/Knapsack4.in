 10 34
 16 5 8 2 8 16 4 8 7 4
 9 6 12 15 25 14 9 15 12 9
