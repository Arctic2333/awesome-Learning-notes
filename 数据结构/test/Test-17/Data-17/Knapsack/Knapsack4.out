Options: 0 0 1 1 1 0 1 1 0 1 
Max Value: 85
